k9mail/src/main/java/com/fsck/k9/preferences/Settings.java

```diff
@@ -274,13 +283,13 @@
      * negligible.
      * </p>
      */
-    public static abstract class SettingsDescription {
+    static abstract class SettingsDescription<A> {
```

Comment:
`PseudoEnumSetting` already had a parameter A, which influenced me there. I agree it should be T :)